tests/pyright/test_file.py

```diff
@@ -1,6 +1,3 @@
-# This is test file for typing,
-# No automatic testing is used at the moment. Just use your type checker and see if it works.
-# Pytest here is used to make sure that runtime behavir matches with type checker expecter errors.
```

Comment:
In addition to this, could you please edit the two tests with comments or docstring indicating that the test itself focused on runtime error, but the typing error is also now tested in `test_typing.py` ? That would be great for maintenance ! Note: it may change the line numbers in the pyright expected output..